src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -296,4 +325,9 @@ function ($type) use ($symbolShortNames) {
 
         return $normalized;
     }
+
+    private function resolveConfiguration()
+    {
+        $this->allowMixed = $this->configuration['allow_mixed'];
```

Comment:
Hi and thanks for the PR :)

Could you move this one-liner in-line in the `configure`-method? than rebase one more time, after that I think it ready (but not 100% my call) to merge

Reply:
Hi, it is fixed and rebased against current master.